### PR TITLE
PP9-12054 - Aggregation filters disabled still show bug fix

### PR DESCRIPTION
**Pull request by tc4rdpp**

### CODE REVIEW

**Comment by tc4rdpp** [comment]
To do: Update Typescript SDK after https://github.com/Picturepark/Picturepark.SDK.TypeScript/pull/416 is merged

File: Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/ClientApp/src/app/components/content-manager/content-manager.component.ts [hunk 1]
```diff
@@ -229,4 +233,21 @@ export class ContentManagerComponent extends PageBase implements OnInit, OnChang
     super.ngOnDestroy();
     this.facade.resetRequestState();
   }
+
+  private filterDisabledAggregators(items: AggregatorBase[]) {
+    return items.filter((item) => item.uiBehavior?.enableFilter);
+  }
+
+  private updateAggregationFilters(state: ContentSearchInputState) {
```
Comment (tc4rdpp) [inline Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/ClientApp/src/app/components/content-manager/content-manager.component.ts:241]: Rename to removeAggregationFilters

Comment (tc4rdpp) [reply Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/ClientApp/src/app/components/content-manager/content-manager.component.ts:245]: Done

**Comment by tc4rdpp** [comment]
> To do: Update Typescript SDK after [Picturepark/Picturepark.SDK.TypeScript#416](https://github.com/Picturepark/Picturepark.SDK.TypeScript/pull/416) is merged

Done

**Review by simobeloz — APPROVED** [review]

**Pull request merged**